### DISCUSSION TITLE
InternLMSpeak : 基于Amphion和InternLM2的音频聊天助手

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#556] [project]
FYI: 
受到 https://github.com/dusty-nv/jetson-containers/tree/master/packages/llm/llamaspeak 项目启发，准备开个坑光速搞一个

Roadmap1:

- [ ]  跑通Amphion TTS ASR
- [ ]  跑通InternLM2 w4a16
- [ ]  组合Amphion 和 InternLM2 w4a16
- [ ] 写前端gradio实现web端适用

Roadmap2:

- [ ] 优化系统实现

Roadmap3:

- [ ] 转出模型为TRT形式，上板子（AGX Orin / AGX Xavier）
- [ ] 做模型图结构优化，适配进TRT-LLM
- [ ] 把TTS和ASM适配进TRT-LLM，并挪上板子

Roadmap4:

- [ ] 重写c++推理部分，对模型进行加速

**Comment by i-am-so-lucky-today** [comment]
这个项目好强
